pkg/kubelet/kubelet.go

```diff
@@ -762,8 +766,13 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 	return node, nil
 }
 
-// registerWithApiserver registers the node with the cluster master.
+// registerWithApiserver registers the node with the cluster master. It is safe
+// to call multiple times, but not concurrently (kl.registrationCompleted is
+// not locked).
```

Comment:
nit: Can you add a comment on why the method is safe to  be called multiple times, but not concurrently?